### PR TITLE
Rename 'core owner' to 'core team member'

### LINKED ISSUE #6: Rename "core owner" to "core team member"?
This is just a suggestion, but I find that I'm a little uncomfortable with the name "core owner". It makes sense given that the role is basically for people who have the "owner" role in the `purescript` org on GitHub, but I worry that it also suggests that we (i.e. the core team) "own" the language, core libraries, etc., which I don't think is quite the model I would like. I think "owner" could perhaps be read in a way that suggests our own personal interests are more important than the interests of the community as a whole, and that people should expect this to be reflected in the direction of the compiler and language. This is not what I want: ideally I think the core team should be seen as people who, as a result of long-term contributions, community engagement, and familiarity with compiler internals, are best placed to understand the current shortcomings of the language/compiler/etc and use that understanding to guide the compiler in a direction that best serves the community. I'm suggesting "core team member" instead because that's something that has been used quite a lot elsewhere too, but I'm open to suggestions - basically I'd just like for it not to contain the word "owner".

### PR DESCRIPTION
**Pull request by JordanMartinez**
Fixes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ exists to maintain and develop:
 
 ## Roles
 
-### Core Owners
+### Core Team Members
 
-The core organization owners lead the development of the language, compiler,
+The core organization team members lead the development of the language, compiler,
```
Comment (hdgarrood) [inline README.md]: I'd shorten this to "The core team lead the development..."

**Review by hdgarrood — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JordanMartinez) [reply README.md]: Done in latest commits.

**Comment by hdgarrood** [comment]
This had enough support in #6 that I think we can consider it approved. Thanks!
